src/DocBlock/Annotation.php

```diff
@@ -187,6 +168,33 @@ public function getTag()
         return $this->tag;
     }
 
+    /**
+     * @return TypeExpression
+     *
+     * @internal until 2.18
```

Comment:
2.19

Reply:
@keradus Do you plan a 2.19 release or shall we write 3.0 here?

Reply:
I assumed 2.19 for now.